firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandler.java

```diff
@@ -162,7 +162,48 @@ public Task<FetchResponse> fetch(long minimumFetchIntervalInSeconds) {
             executor,
             (cachedFetchConfigsTask) ->
                 fetchIfCacheExpiredAndNotThrottled(
-                    cachedFetchConfigsTask, minimumFetchIntervalInSeconds));
+                    cachedFetchConfigsTask, minimumFetchIntervalInSeconds, false));
+  }
+
+  /**
+   * Starts fetching configs from the Firebase Remote Config server.
```

Comment:
I might include `without setting the ETag header` up here just cause it's the main distinction